apps/src/templates/projects/FeaturedProjectsTable.jsx

```diff
@@ -120,7 +112,8 @@ const typeFormatter = (type) => {
 
 class FeaturedProjectsTable extends React.Component {
   static propTypes = {
-    projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
+    projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired,
+    tableVersion: PropTypes.oneOfType(['currentFeatured', 'archiveFeatured']).isRequired
```

Comment:
it'd be great to make these strings exportable constants to avoid spelling mistakes